### PR TITLE
Add issue assignation policy + warning about meilisearch-js priority in pre-release

### PR DESCRIPTION
**Pull request by curquiza**
Should be merged with both approval of @bidoubiwa and @alallema

### CODE REVIEW

File: resources/issues.md [hunk 1]
```diff
@@ -22,3 +22,13 @@ The following labels help us categorize the issues:
...
+
+## Issue assignation policy
+
+We prefer not assigning external people to our issues.
+
+Why? It often occurs that people ask to be assigned and never come back, which discourages the volunteer contributors to open a PR to fix the issue.
+
+We will accept and merge the first PR that fixes correctly and well implements the issue following the contribution guidelines of the corresponding repository.
+
+We are looking forward to seeing your contribution, and no need to wait for an assignation to start it! 👍
```
Comment (bidoubiwa) [inline resources/issues.md:34]:
```suggestion
We are looking forward to your contributions, no need to wait for an assignation to start! 👍
```

File: resources/pre-release-week.md [hunk 2]
```diff
@@ -46,6 +46,10 @@ The script automates the following tasks:
 
 ## 💻 Coding
 
+⚠️ [`meilisearch-js`](https://github.com/meilisearch/meilisearch-js) and [`instant-meilisearch`](https://github.com/meilisearch/instant-meilisearch/) implementations should start as soon as possible since:
+- they are related to the [`mini-dashboard`](https://github.com/meilisearch/mini-dashboard).
```
Comment (bidoubiwa) [inline resources/pre-release-week.md:50]:
```suggestion
- they are dependencies of the [`mini-dashboard`](https://github.com/meilisearch/mini-dashboard).
```

Should we mention: 
- docs-searchbar
- gatsby
- firebase
- Demos (devrel and playgrounds)
- Strapi
- (Saas in the future?)

?

File: resources/pre-release-week.md [hunk 3]
```diff
@@ -46,6 +46,10 @@ The script automates the following tasks:
...
+- they can make us report a bad design choice of the API when using MeiliSearch in front-end.
```
Comment (bidoubiwa) [inline resources/pre-release-week.md:51]:
```suggestion
- they may raise bad design choices or relevancy issues of the MeiliSearch API when used in front-end.
```

**Review by bidoubiwa — CHANGES_REQUESTED** [review]

File: resources/pre-release-week.md (hunk 2 above)
Comment (curquiza) [reply resources/pre-release-week.md:50]: We should not list the dependencies here. We are in the pre-release context, that's why the mini-dashboard is mentioned.

Comment (curquiza) [reply resources/pre-release-week.md]: I plan to do a guide gathering all the dependencies in integration team (Laura asked for it anyway)

**Review by alallema — APPROVED** [review]
Agree with the modifications requested by @bidouwa.
The rest of it LGTM!

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥

**Pull request merged**